crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -534,6 +536,53 @@ impl Output for CliOutput {
         );
     }
 
+    fn print_job_output(&self, job: JobDetail, task_selector: &Selector, output_stream: &OutputStream) -> anyhow::Result<()> {
+        fn read_stream(task_state: &JobTaskState, output_stream: &OutputStream) {
+            let (_, stdout, stderr) = get_task_paths(task_state);
+            let stdin = match output_stream {
+                OutputStream::Stdout => {stdout}
+                OutputStream::Stderr => {stderr}
+            };
+
+            let file = File::open(stdin).expect("Unable to open file");
+            let mut reader = BufReader::new(file);
+            for line in reader.lines() {
+                let line = line.expect("Unable to read line");
+                println!("{}", line);
+            }
+        }
+
+        let JobDetail {
```

Comment:
Only `tasks` are needed, so there's no need to destructure `JobDetail` here. Just access `job.tasks`. Destructuring is used in places where we want to be notified when a new attribute is added to some struct.